gradle.properties

```diff
@@ -11,4 +11,4 @@ rsApi=3.0.0
 faker=1.0.2
 swagger=2.1.13
 jacksonVersion=2.13.1
```

Comment:
There is inconsistency in var naming. I would suggest sticking to `xxxVersion` (can be done in separate PR)

Reply:
Here we align names and versions with EDC